assets/svelte/components/SidebarSection.svelte

```diff
@@ -188,14 +188,24 @@
               </div>
             </div>
           {:else}
-            <input
-              type="text"
-              class="w-full py-1 px-2 mt-5 bg-slate-100 border-slate-100 rounded-md leading-6 text-sm"
-              {placeholder}
-              value={astNode}
-              on:keydown={handleKeydown}
-              on:change={(e) => updateNodeContents(e, idx)}
-            />
+            {#if large}
```

Comment:
If `large={true}` do we even need this conditional?

Reply:
`<SidebarSection>` is not always called with `large={true}`